src/schematron/iso-schematron/iso_schematron_skeleton_for_saxon.xsl

```diff
@@ -516,7 +516,7 @@ which require a preprocess.
   which I find a bit surprising but anyway I'll use the longr faster version.
 -->
 <xsl:variable name="context-xpath">
-  <xsl:if test="$attributes='true' and parent::node() ">@*|</xsl:if>
+  <xsl:if test="$attributes='true'">@*|</xsl:if>
```

Comment:
This concerns all the Schematron changes made in `iso_schematron_skeleton_for_saxon.xsl`: if they are not strictly needed to run the ant build file, I suggest to wait for these changes to be integrated in the [official Schematron repository](https://github.com/schematron/schematron), I saw you submitted a pull request there too.